spec/01-unit/01-db/01-schema/11-declarative_config/02-process_auto_fields_spec.lua

```diff
@@ -375,6 +375,7 @@ describe("declarative config: process_auto_fields", function()
                     protocols = { "grpc", "grpcs", "http", "https" },
                     config = {
                       hide_credentials = false,
+                      realm = "service",
```

Comment:
Probably not needed, as this is the default in the schema

Reply:
Unfortunately without this the test fails and I guess it's ok since as far as I understand it this test in [line#306](https://github.com/Kong/kong/pull/11795/files#diff-d67bdb1532159f004104b149195dba283e15ebdb56b539a18b76472b85275dd5R306) dumps schema with DecK and then validates against this.
From what I see this is what happens after running `deck gateway dump -o kong.yaml`
<img width="402" alt="Screenshot 2023-12-05 at 09 27 46" src="https://github.com/Kong/kong/assets/5149215/e60f14df-7789-4d0f-9cf0-6b9d4ccad201">